### DISCUSSION TITLE
Open graph debugger: 500 x-vercel-error: INTERNAL_SERVER_ERROR on Vercel

### DISCUSSION THREAD

**Discussion by piotrgrundas** [vercel/community#6384] [Help]
### Page to Investigate

[https://projectluna.pl/](https://projectluna.pl/)

### Steps to Reproduce

Run a curl command:

```
$ curl -v --compressed -H "Range: bytes=0-524288" -H "Connection: close" -A "facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)" "https://projectluna.pl"      ─╯
*   Trying 76.76.21.21:443...
* Connected to projectluna.pl (76.76.21.21) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (IN), TLS handshake, Server hello (2):
* (304) (IN), TLS handshake, Unknown (8):
* (304) (IN), TLS handshake, Certificate (11):
* (304) (IN), TLS handshake, CERT verify (15):
* (304) (IN), TLS handshake, Finished (20):
* (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=projectluna.pl
*  start date: Mar 11 16:31:57 2024 GMT
*  expire date: Jun  9 16:31:56 2024 GMT
*  subjectAltName: host "projectluna.pl" matched cert's "projectluna.pl"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://projectluna.pl/
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: projectluna.pl]
* [HTTP/2] [1] [:path: /]
* [HTTP/2] [1] [user-agent: facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)]
* [HTTP/2] [1] [accept: */*]
* [HTTP/2] [1] [accept-encoding: deflate, gzip]
* [HTTP/2] [1] [range: bytes=0-524288]
> GET / HTTP/2
> Host: projectluna.pl
> User-Agent: facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)
> Accept: */*
> Accept-Encoding: deflate, gzip
> Range: bytes=0-524288
> Connection: close
>
< HTTP/2 307
< cache-control: public, max-age=0, must-revalidate
< content-type: text/plain
< date: Sat, 20 Apr 2024 12:51:35 GMT
< link: <https://projectluna.pl/pl>; rel="alternate"; hreflang="pl", <https://projectluna.pl/pl-pl>; rel="alternate"; hreflang="pl-pl", <https://projectluna.pl/>; rel="alternate"; hreflang="x-default"
< location: /pl
< server: Vercel
< set-cookie: NEXT_LOCALE=pl; Path=/; Expires=Sun, 20 Apr 2025 12:51:35 GMT; Max-Age=31536000; SameSite=lax
< strict-transport-security: max-age=63072000
< x-vercel-id: arn1::stqhs-1713617495669-e068c116ef4a
<
Redirecting...
* Connection #0 to host projectluna.pl left intact
```

In the response we can see http 500 `x-vercel-error: INTERNAL_SERVER_ERROR`.

-  Using [open graph debugger](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fprojectluna.pl) I also see `URL returned a bad HTTP response code.` (500).
- Vercel fn logs do not show any errors.
- Building the project locally and running the above curl through ngrok gives me no errors.

Any help would be appreciated!

**Comment by piotrgrundas** [comment]
Bump

**Comment by piotrgrundas** [comment]
@amyegan 
Could I ask to take a look?

**Reply by versecafe** [reply]
@piotrgrundas can you share what the actual server side error logs look like?

**Comment by piotrgrundas** [comment]
The problem is that there is nothing there.
@versecafe  here u go:

![image](https://github.com/vercel/community/assets/5078402/a5a9b179-c1a7-4be9-9625-5aa6942af80f)
![image](https://github.com/vercel/community/assets/5078402/b91a16da-8b01-4e0d-811f-5e224cae59f8)
![image](https://github.com/vercel/community/assets/5078402/6d7b23e0-8509-44bf-8464-f9fa2106d541)
![image](https://github.com/vercel/community/assets/5078402/7c1f8e2a-9b8f-46c3-a873-72a5c261c2ce)
![image](https://github.com/vercel/community/assets/5078402/0b62b591-0315-4ee6-b838-39b4da9591f4)

**Comment by piotrgrundas** [comment]
Can any one rom Vercel look at this issue? 
This is so frustrating 😞

**Reply by amyegan** [reply]
Hey @piotrgrundas. I've not able to see a 500 error with it myself. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we could use to investigate further?

**Comment by piotrgrundas** [comment]
Closing - it started to work without any change on my side.

[a comment by an automated account is not shown]
